MAUI/MauiAppBasic/MSALClient/PCAWrapper.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+
+namespace MauiAppBasic.MSALClient
+{
+    /// <summary>
+    /// This is a wrapper for PublicClientApplication. It is singleton and can be utilized by both application and the MAM callback
+    /// </summary>
+    public class PCAWrapper
+    {
+        /// <summary>
+        /// This is the singleton used by consumers. Since PCAWrapper constructor does not have perf or memory issue, it is instantiated directly.
+        /// </summary>
+        public static PCAWrapper Instance { get; private set; } = new PCAWrapper();
+
+        /// <summary>
+        /// Instance of PublicClientApplication. It is provided, if App wants more customization.
+        /// </summary>
+        internal IPublicClientApplication PCA { get; }
+
+        /// <summary>
+        /// This will determine if the Interactive Authentication should be Embedded or System view
+        /// </summary>
+        internal bool UseEmbedded { get; set; } = false;
+
+        // ClientID of the application in (sample-testing.com)
+        internal const string ClientId = "4b706872-7c33-43f0-9325-55bf81d39b93"; // TODO - Replace with your client Id. And also replace in the AndroidManifest.xml

```

Comment:
All the needed config changes should be listed in a readme.

Reply:
Added new MauiApps.md file